GoogleForCreators/web-stories-wp
# Layer Panel: Add a max draggable height

## Context

FIXIT week

## Summary

The layer panel can no longer be dragged past the last layer

## Relevant Technical Choices

n/a

## To-do

n/a

## User-facing changes

|Before|After|
|--|--|
|![before](https://user-images.githubusercontent.com/22185279/117172592-2d799680-ad89-11eb-93fc-d77af9f60fed.gif)|![after](https://user-images.githubusercontent.com/22185279/117172607-30748700-ad89-11eb-9a2b-ee204059bc22.gif)|

## Testing Instructions

1. Go to editor (using a template is nice since there will be many layers already
2. Drag the layer panel up and down. Should be collapsible but shouldn't be draggable past the 'background' layer

### QA






### UAT






## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #6600
